cmake/Reprojucer.cmake

```diff
@@ -1625,6 +1657,21 @@ function(__set_bundle_properties target_name extension)
 endfunction()
 
 
+function(__set_install_after_build target_name destination)
+
+  install(TARGETS ${target_name} COMPONENT ${target_name} DESTINATION ${destination})
```

Comment:
This can conflict with user-defined `install` rules, so we need to find another approach.